src/styles/index.scss

```diff
@@ -1,3 +1,109 @@
-body {
+$main-color: #060b35;
+$secondary-color: #616070;
+
+* {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
```

Comment:
according to [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) there should be 1 file for each BEM block - please move create card file and move it there